Day24/Solution.swift

```diff
@@ -0,0 +1,11 @@
+func removeDuplicates(head: Node?) -> Node? {
+    var current = head
+    while current?.next != nil {
+        if current?.data == current?.next?.data {
```

Comment:
your solution is timing out when I'm submitting it on hackerrank, I request you to update it and test on hackerrank.